src/index.ts

```diff
@@ -501,7 +526,291 @@ app.get('/givDumpers', async (req: Request, res: Response) => {
 })
 
 
+app.get(`/calculate-updated`,
+  async (req: Request, res: Response) => {
+    try {
+      console.log('start calculating')
+      const {
+        download, roundNumber,
+        maxAddressesPerFunctionCall,
+        givRelayerAddress,
```

Comment:
We should not be getting this anymore from the API call - this is not how it is current deployment